.gitpod.yml

```diff
@@ -1,4 +1,4 @@
-image: mcr.microsoft.com/dotnet/sdk:6.0
+image: gitpod/workspace-dotnet:latest

```

Comment:
Wait are we saying that this also fails with `mcr.microsoft.com/dotnet/sdk:7.0`? 

If so are we are risk of this image drifting from .NET v7?

Reply:
Great question, didn't think to check

Reply:
Yep that solves the issue - let's go with that, and I'll raise another PR for the others